crates/ruff_diagnostics/src/edit.rs

```diff
@@ -4,13 +4,13 @@ use serde::{Deserialize, Serialize};
 
 #[derive(Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
 #[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
-pub struct Fix {
+pub struct Edit {
     pub content: String,
     pub location: Location,
     pub end_location: Location,
 }
 
-impl Fix {
+impl Edit {
     pub const fn deletion(start: Location, end: Location) -> Self {
```

Comment:
Unrelated to this PR but I wonder if it would be more appropriate to use an `enum` instead

```rust
enum Edit {
  Deletion { start: Location, end: Location },
  Insertion { text: String, at: Location },
  Replacement { text: String, start: Location, end: Location }
}
```

Or

```rust
struct Edit {
	kind: EditKind,
	at: Location
}

enum EditKind {
	Deletion { end: Location },
	Insertion { text: String },
	Replacement { with: String, end: Location }
}
```